src/lib.rs

```diff
@@ -292,6 +267,19 @@ mod arch {
     }
 }
 
+fn sudo_command(sudo: Option<&Option<String>>, command: &str) -> Command {
```

Comment:
I'd like to see the following changes:

* Invert the order of the arguments
* Change the logic inside to do an early return for the simple case, reducing rightward drift:
  ```rust
  let sudo = match sudo {
    Some(sudo) => sudo,
    None => return Command::new(command),
  };
  
  ..
  ```